src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,61 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import Text from './Text';
+import colors from '../styles/colors';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+
+const propTypes = {
+
+    isDefaultChatRoom: PropTypes.bool,
+    chatUsers: PropTypes.arrayOf(PropTypes.object),
+    ...withLocalizePropTypes,
+
+};
+
+const defaultProps = {
+    chatUsers: [],
+    isDefaultChatRoom: false,
+};
+
+const ChatBeginingText = ({isDefaultChatRoom, chatUsers, translate}) => (
+    <Text style={[styles.mt3, styles.w70, styles.textAlignCenter]}>
+        <Text style={[{color: colors.dark}]}>
```

Comment:
@roryabraham thoughts on this kind of inline style? Does all of our text use the correct dark color by default or no?

Reply:
i don't think so.